protected/humhub/modules/admin/controllers/ApprovalController.php

```diff
@@ -77,28 +80,59 @@ public function beforeAction($action)
         return parent::beforeAction($action);
     }
 
+    /**
+     * @return string
+     * @throws \Throwable
+     * @throws \yii\base\InvalidConfigException
+     */
     public function actionIndex()
     {
         $searchModel = new UserApprovalSearch();
         $dataProvider = $searchModel->search(Yii::$app->request->queryParams);
 
+        // Get available profile fields for screen options
+        $availableProfileFields = ProfileField::find()
+            ->joinWith('category')
+            ->orderBy([
+                'profile_field_category.sort_order' => SORT_ASC,
+                'profile_field.sort_order'=>SORT_ASC
+            ])
+            ->where(['profile_field.show_at_registration' => true])
+            ->andWhere(['not', ['profile_field.internal_name' => ['firstname', 'lastname']]])
+            ->all();
+
+        // Get or set screen options
+        $userSettings = Yii::$app->settings->user(Yii::$app->user->identity);
+        $screenProfileFieldsId = $userSettings->getSerialized(self::USER_SETTINGS_SCREEN_KEY, []);
```

Comment:
I would recommend:

``` 
        /** @var Module $module */
        $module = Yii::$app->getModule('admin');
        $module->settings->user()->getSerialized(self::USER_SETTINGS_SCREEN_KEY, []);
``` 

This would scope the settings into the admin module.